src/Routes.js

```diff
@@ -97,7 +100,7 @@ export const AppRoutes = () => {
         <Route
           path="/workloads"
           element={
-            workloadsEnabled ? (
+            workloadsEnabled && beta ? (
```

Comment:
This will prevent the workloads feature from working in stage-stable environment. Is it what we are aiming to do?

Reply:
Yes, this is going to be inevitable if we want to do a production preview release without showing it in prod stable :(

Reply:
The plan is to remove feature flag and these checks as soon as we release to prod stable

Reply:
> Yes, this is going to be inevitable if we want to do a production preview release without showing it in prod stable :(

I believe this is doable with:
```
!(chrome.isProd() && chrome.isBeta()) && workloadsEnabled
```